Description: do not build the md2html example.
 The author expressed it's only an example code.
 It is currently not being installed as an example, so
 simply avoid building it.
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: not-needed

---
 CMakeLists.txt |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,4 +56,5 @@
 include(GNUInstallDirs)
 
 add_subdirectory(src)
-add_subdirectory(md2html)
+# Do not build the example.
+#add_subdirectory(md2html)
